### PR TITLE
Update illustrations on openstack page

### LINKED ISSUE #6590: [ILLUSTRATIONS UPDATE] /openstack
[copydoc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit)

**Hero**

- [x] replace illustration with this one: https://assets.ubuntu.com/v1/669390e0-openstack-cloud.svg

- [x] `add p-strip--suru-topped`

**Learn more about OpenStack**

- [x] WEBINAR: https://assets.ubuntu.com/v1/6e184942-Webinar.svg

- [x] WEBINAR: https://assets.ubuntu.com/v1/6e184942-Webinar.svg

- [x] EBOOK: https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update illustrations on /openstack

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the updated illustrations match those outlined in [the issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/6590)

## Issue / Card

Fixes #6590

### CODE REVIEW

File: templates/openstack/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip is-deep is-bordered">
+<section class="p-strip--suru-topped is-deep is-bordered">
   <div class="row">
```
Comment (pmahnke) [inline templates/openstack/index.html]: I think this will need a `u-vertically-center`

Comment (pmahnke) [inline templates/openstack/index.html]:
```suggestion
  <div class="row u-vertically-center">
```

Comment (pmahnke) [reply templates/openstack/index.html]: I think this will need a `u-vertically-center`

**Comment by anasereijo** [comment]
- [ ] hero: vertically align the illustration
Screenshot 2020-02-10 at 09 10 27

**Comment by steverydz** [comment]
Thanks @pmahnke and @anasereijo. I've centered the hero image.

**Pull request merged**